server/services/reportsService.ts

```diff
@@ -22,38 +30,53 @@ export = function createReportsService(
     )
   }
 
-  const isUnallocated = (pd: OffenderDetail) => {
+  private static isUnallocated(pd: OffenderDetail) {
     const results = pd.offenderManagers.some(
       (om) => om.active === true && (om.staff.unallocated === true || om.staff.code.endsWith('U'))
     )
     return results
   }
 
-  const getEligibleLicencesWithAddressOrCasLocation = async (): Promise<Case[]> => {
+  private static getPduDescription(probationDetail: OffenderDetail): string {
+    if (!probationDetail) return ''
+    const activeOffenderManager = probationDetail.offenderManagers.find((om) => om.active)
+    if (!activeOffenderManager) return ''
+    const { description } = activeOffenderManager.probationArea
+    return description || ''
+  }
+
+  private static getComName(probationDetail: OffenderDetail): string {
+    if (!probationDetail || ReportsService.isUnallocated(probationDetail)) return ''
+    const activeOffenderManager = probationDetail.offenderManagers.find((om) => om.active)
+    const { forenames, surname } = activeOffenderManager.staff
```

Comment:
I think we might be missing a check for if `activeOffenderManager` is null here (if the `find` doesn't return anything? Presumably it always should, but you've accounted for it in `getPduDescription` above, so I assume there's a possibility.

Reply:
so the check on line 49 will return an empty string if the `probationDetail` exists but is unallocated so if we pass that check I think there will always be an `activeOffenderManager` present? Is that a fair assumption? 😅

Reply:
Think that is a valid assumption to have.